### DISCUSSION TITLE
octoprint bed visualizer gcode E3V2 help

### DISCUSSION THREAD

**Discussion by whitefiretiger** [mriscoc/Ender3V2S1#828] [Q&A]
i could use some help figuring out what gcode to use for octoprint bed visualizer.
Ender3 V2 and every gcode I've found and tried fails either pulls current saved on printer or drops probe and doesn't move after homing

**Comment by MarkforU** [comment]
I would be happy to see the current mesh with tilt. I have the plug-in installed, but yet to configure it.

**Comment by mriscoc** [comment]
Allways check the Wiki: https://github.com/mriscoc/Ender3V2S1/wiki/Octoprint#bed-level-visualizer-support
